src/components/Pages/Topic.js

```diff
@@ -43,20 +41,20 @@ const Topic = ({
         <div className="col-xs-12 coa-TopicPage__tile-group">
           {!!topLinks.length && (
             <TileGroup
-              text={intl.formatMessage(i18n.topServices)}
+              title={intl.formatMessage(i18n.topServices)}
               tiles={topLinks}
             />
           )}
           {!!otherLinks.length && (
             <TileGroup
-              text={intl.formatMessage(i18n.allServices)}
+              title={intl.formatMessage(i18n.allServices)}
```

Comment:
was this fixing a bug, or is it related to the restructuring? TileGroup doesn't have a text prop in master, so im thinking its a bug fix

Reply:
Yes, if we look at master/prod we'll see that the Topic pages aren't showing "Top services" or "All services" anywhere. This was just a bug find/fix.